collector/meminfo_bsd.go

```diff
@@ -92,19 +87,18 @@ func (c *meminfoCollector) Update(ch chan<- prometheus.Metric) (err error) {
 		}
 	}
 
-	log.Debugf("Set node_mem: %#v", pages)
 	for k, v := range pages {
-		if _, ok := c.metrics[k]; !ok {
-			c.metrics[k] = prometheus.NewGauge(prometheus.GaugeOpts{
-				Namespace: Namespace,
-				Subsystem: memInfoSubsystem,
-				Name:      k,
-				Help:      k + " from sysctl()",
-			})
-		}
 		// Convert metrics to kB (same as Linux meminfo).
 		c.metrics[k].Set(float64(v) * float64(size))
 		c.metrics[k].Collect(ch)
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName(Namespace, memInfoSubsystem, k),
+				k+" from sysctl()",
+				nil, nil,
+			),
+			prometheus.UntypedValue, v,
```

Comment:
I changed this to `Untyped` here because I wasn't sure whether `vm.stats.vm.v_swappgsin` and `vm.stats.vm.v_swappgsout` are counters. Can anyone confirm?